pkg/tsdb/azuremonitor/azuremonitor.go

```diff
@@ -19,6 +19,7 @@ import (
 	"github.com/grafana/grafana-plugin-sdk-go/backend/log"
 	"github.com/grafana/grafana-plugin-sdk-go/backend/resource/httpadapter"
 
+	ngalertmodels "github.com/grafana/grafana/pkg/services/ngalert/models"
```

Comment:
We can't import this as it's core Grafana code, we'll have to hardcode the header

Reply:
changed to a variable within the function :)